src/lib.rs

```diff
@@ -74,6 +74,20 @@
 //! and Node.js environments are supported, invoking the methods
 //! [described above](#supported-targets) using the [`wasm-bindgen`] toolchain.
 //!
+//! To enable the `js` Cargo feature, add the following to the `dependencies`
+//! section in your `Cargo.toml` file:
+//! ```toml
+//! [dependencies]
+//! getrandom = { version = "0.2", features = ["js"] }
+//! ```
+//!
+//! This can be done even if `getrandom` is not a direct dependency. Cargo
+//! allows crates to enable features for indirect dependencies.
+//!
+//! This feature should only be enabled for binary, test, or benchmark crates.
+//! Library crates should generally not enable this feature, leaving such a
+//! decision to *users* of their library.
```

Comment:
It may be worth to also add that library crates should not introduce their own `js` features which simply enable `getrandom`'s one.

Reply:
Good idea, done!